include/lsst/afw/table/io/FitsReader.h

```diff
@@ -15,135 +17,85 @@ namespace lsst { namespace afw { namespace table { namespace io {
  *  FitsReader itself provides the implementation for reading standard FITS binary tables
```

Comment:
GitHub won't let me comment on line 15 above since it's not part of these changes, but note that this is no longer "A Reader subclass", so the documentation needs tweaking.